otter/test/log/test_log.py

```diff
@@ -158,6 +162,16 @@ def test_failure_logging(self):
         self.observer.assert_called_once_with(
             {'message': (SameJSON({'foo': str(failure)}),)})
 
+    def test_message_is_concatenated(self):
+        """
+        message tuple in event is concatenated before passing on
+        """
+        eventDict = {'message': ('mine',)}
```

Comment:
This doesn't really test if the message tuple is concatenated -- maybe you should add a second element here